src/models/player.js

```diff
@@ -20,16 +20,18 @@ export default class Player extends BaseModel {
     return this.data.attributes.createdAt;
   }
 
-  get stats() {
+  get skillTier() {
     const stats = this.raw.attributes.stats;
-    
-    const skillTier = skillTiers.find(tier => tier.serverName === stats.skillTier);
-    const karmaLevel = karma.find(k => k.serverName === stats.karmaLevel);
+    return skillTiers.find(tier => tier.serverName === stats.skillTier) || "Unknown";
```

Comment:
This looks good. Instead of "Unknown", let's use null as it'll be easier to make a conditional statement out of. Likewise with karmaLevel below.

Reply:
Fixed. It was actually a misspelling as the class method is camelcase